src/components/MapArea.tsx

```diff
@@ -186,54 +191,51 @@ export default function MapArea({ selectedTrail, searchResult }: Props) {
   }, [searchResult])
 
   const addSearchMarkersToMap = (markerData: any) => {
-    const markers: Marker[] = [];
-    markerData.forEach((feature: any) => {
-      const marker = new mapboxgl.Marker()
-        .setLngLat(feature.center)
-        .setPopup(new mapboxgl.Popup({ offset: 25 }).setText(`${feature.place_name}`))
-        .addTo(map.current as Map)
-      markers.push(marker);
+    // Add 'place_name' key and its value to the properties object
+    // This allows the marker event to have access to the value of 'place_name'
+    markerData.forEach((marker: any) => {
+      marker.properties['place_name'] = marker.place_name;
     })
-    setPOIMarkers(markers);
-    // console.log("markerData", markerData);
-    // console.log("getsource", map.current?.getSource("markers"));
 
-    // if (!map.current?.getSource("markers")) {
-    //   map.current?.addSource("markers", {
-    //     type: "geojson",
-    //     data: {
-    //       type: "FeatureCollection",
-    //       features: [...markerData],
-    //     },
-    //   });
-    //   map.current?.addLayer({
-    //     id: "markers",
-    //     type: "circle",
-    //     source: "markers",
-    //     paint: {
-    //       "circle-radius": 10,
-    //       "circle-color": "#007cbf",
-    //     },
-    //   });
-    // }
+    map.current?.loadImage(
+      "https://upload.wikimedia.org/wikipedia/commons/thumb/8/88/Map_marker.svg/780px-Map_marker.svg.png",
```

Comment:
what if this url changes? your app will no longer show markers. Better to have this marker in your assets